docs/examples/allow-all.yaml

```diff
@@ -0,0 +1,48 @@
+# This policy configures all options to allow all possible values.
+apiVersion: policy.cert-manager.io/v1alpha1
+kind: CertificateRequestPolicy
+metadata:
+  name: allow-all
+spec:
+  allowed:
+    commonName: {value: "*"}
+    dnsNames: {values: ["*"]}
+    ipAddresses: {values: ["*"]}
+    uris: {values: ["*"]}
+    emailAddresses: {values: ["*"]}
+    isCA: true
+    usages:
```

Comment:
Wow! This is a long list! Have we ever considered adding wildcard support to this field?